0.60.0 broken can’t open init

/bin/sh: can't open '/init': Permission denied

Here are the logs :

2025-11-19T08:45:55+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:46:05+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:46:28+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:46:37+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:46:38+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:46:44+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:47:02+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:47:16+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:47:21+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
2025-11-19T08:47:26+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:159: sending offer with serial: xxxxxxxxxx
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service netbird: stopping
[08:47:34] INFO: [32mService NetBird Client exited with code 256 (by signal 15)[0m
s6-rc: info: service netbird successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service base-addon-log-level: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service base-addon-log-level successfully stopped
s6-rc: info: service base-addon-banner: stopping
s6-rc: info: service base-addon-banner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
2025-11-19T08:47:34+01:00 INFO client/cmd/root.go:196: shutdown signal received
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:114: stop listening for remote offers and answers
2025-11-19T08:47:34+01:00 INFO client/internal/wg_iface_monitor.go:58: Interface monitor: stopped for wt0
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:114: stop listening for remote offers and answers
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:114: stop listening for remote offers and answers
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:114: stop listening for remote offers and answers
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/handshaker.go:114: stop listening for remote offers and answers
2025-11-19T08:47:34+01:00 INFO client/internal/connect.go:299: ensuring wt0 is removed, Netbird engine context cancelled
2025-11-19T08:47:34+01:00 INFO client/internal/engine.go:293: Network monitor: stopped
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-1.relay.netbird.io:yyy] shared/relay/client/client.go:597: closing all peer connections
2025-11-19T08:47:34+01:00 WARN client/internal/engine.go:528: WireGuard interface monitor: wg interface monitor stopped: context canceled
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-0.relay.netbird.io:yyy] shared/relay/client/client.go:597: closing all peer connections
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-1.relay.netbird.io:yyy] shared/relay/client/client.go:370: start to Relay read loop exit
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-1.relay.netbird.io:yyy] shared/relay/client/client.go:605: waiting for read loop to close
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-1.relay.netbird.io:yyy] shared/relay/client/client.go:607: relay connection closed
2025-11-19T08:47:34+01:00 WARN [relay: rels://streamline-es-mad1-1.relay.netbird.io:yyy] shared/relay/client/client.go:588: relay connection was already marked as not running
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-0.relay.netbird.io:yyy] shared/relay/client/client.go:370: start to Relay read loop exit
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-0.relay.netbird.io:yyy] shared/relay/client/client.go:605: waiting for read loop to close
2025-11-19T08:47:34+01:00 INFO [relay: rels://streamline-es-mad1-0.relay.netbird.io:yyy] shared/relay/client/client.go:607: relay connection closed
2025-11-19T08:47:34+01:00 WARN [relay: rels://streamline-es-mad1-0.relay.netbird.io:yyy] shared/relay/client/client.go:588: relay connection was already marked as not running
2025-11-19T08:47:34+01:00 INFO client/internal/routemanager/manager.go:307: Routing cleanup complete
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:228: close peer connection
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:262: peer connection closed
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:228: close peer connection
2025-11-19T08:47:34+01:00 WARN [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/worker_relay.go:124: failed to close relay connection: use of closed network connection
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:262: peer connection closed
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:228: close peer connection
2025-11-19T08:47:34+01:00 WARN [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/worker_relay.go:124: failed to close relay connection: use of closed network connection
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:262: peer connection closed
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:228: close peer connection
2025-11-19T08:47:34+01:00 WARN [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/worker_relay.go:124: failed to close relay connection: use of closed network connection
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:262: peer connection closed
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:228: close peer connection
2025-11-19T08:47:34+01:00 INFO [peer: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=] client/internal/peer/conn.go:262: peer connection closed
2025-11-19T08:47:34+01:00 ERRO client/iface/udpmux/universal.go:98: error while reading packet: shared socked stopped
2025-11-19T08:47:34+01:00 INFO client/iface/iface.go:309: interface wt0 has been removed
2025-11-19T08:47:34+01:00 INFO client/internal/engine.go:353: stopped Netbird Engine
2025-11-19T08:47:34+01:00 INFO client/internal/connect.go:308: stopped NetBird client
2025-11-19T08:47:34+01:00 INFO shared/signal/client/worker.go:51: Message worker stopping due to context cancellation
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied
/bin/sh: can't open '/init': Permission denied

--- a/netbird/config.yaml
+++ b/netbird/config.yaml
@@ -19,11 +19,8 @@ privileged:
   - NET_RAW
   - BPF
 map:
-  - type: homeassistant_config
-    read_only: false
-  - type: addon_config
-    read_only: false
-    path: /var/lib/netbird
+  - homeassistant_config:rw
+  - addon_config:rw
 options:
   admin_url: ""
   management_url: ""

--- a/netbird/rootfs/etc/s6-overlay/s6-rc.d/netbird/run
+++ b/netbird/rootfs/etc/s6-overlay/s6-rc.d/netbird/run
@@ -11,19 +11,9 @@ declare value
 
 # Get the options configured in HASS GUI
 readonly CONFIG_OLD_PATH=/homeassistant/netbird/config.json
-readonly CONFIG_PATH=/var/lib/netbird/config.json
-readonly CONFIG_PROFILE_PATH=/var/lib/netbird/default.json
+readonly CONFIG_PATH=/config/config.json
 
-# Migrate old configuration files to new location (one-time migration)
-if [ -f "${CONFIG_OLD_PATH}" ] && [ ! -f "${CONFIG_PROFILE_PATH}" ]; then
-    bashio::log.info "Migrating configuration from old location..."
-    mv "${CONFIG_OLD_PATH}" "${CONFIG_PATH}" && bashio::log.info "Migration step 1 complete" || bashio::log.error "Migration step 1 failed: could not move ${CONFIG_OLD_PATH} to ${CONFIG_PATH}"
-fi
-
-if [ -f "${CONFIG_PATH}" ] && [ ! -f "${CONFIG_PROFILE_PATH}" ]; then
-    bashio::log.info "Setting up NetBird profile configuration..."
-    mv "${CONFIG_PATH}" "${CONFIG_PROFILE_PATH}" && bashio::log.info "Migration step 2 complete" || bashio::log.error "Migration step 2 failed: could not move ${CONFIG_PATH} to ${CONFIG_PROFILE_PATH}"
-fi
+[ -f "${CONFIG_OLD_PATH}" ] && mv "${CONFIG_OLD_PATH}" "${CONFIG_PATH}"
 
 admin_url="$(bashio::config 'admin_url')"
 management_url="$(bashio::config 'management_url')"
@@ -34,6 +24,7 @@ rosenpass_permissive="$(bashio::config 'rosenpass_permissive')"
 log_level="$(bashio::config 'log_level')"
 
 options+=(--foreground-mode)
+options+=(--config "${CONFIG_PATH}")
 options+=(--log-file console)
 
 if [ "${admin_url}" = "" ]; then

